packages/aws-cdk-lib/aws-lambda/lib/function-url.ts

```diff
@@ -59,6 +59,23 @@ export enum HttpMethod {
   ALL = '*',
 }
 
+
+/**
+ * The invoke modes for a function url
+ */
+export enum FunctionUrlInvokeMode {
+
+  /**
+   *  Invocation results are available when the payload is complete.
+   */
+  BUFFERED = 'BUFFERED',
+
+  /**
+   * Your function streams payload results as they become available.
+   */
+  STREAMED = 'STREAMED',
```

Comment:
This should probably be [`RESPONSE_STREAM`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-url.html#cfn-lambda-url-invokemode)